randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+        # Replaces Vault bomb slot seal scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;These Bomb Slot covers are able to be destroyed by &main-color=#FF3333;Power Bombs"
+            "&main-color=#89D6FF;.\n\nAdditionally, these Bomb Slots are also able to be activated by Power Bombs."
+        ),
+    ],
+    "1354393071": [
+        # Watery Hall PB Sandstone Tip
+        # Replaces Watery Hall sandstone scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures, "
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "3507547160": [
+        # Lava Lake PB Sandstone + No-Respawn Tip
+        # Replaces Lava Lake sandstone scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures, "
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;.\n\n"
+            "Additionally, in most game versions, these rocks will not reappear after being destroyed."
+        ),
+    ],
+    "3269646811": [
+        # Reflecting Pool PB Talloric Alloy Tip
+        # Replaces Reflecting Pool lid scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Talloric Alloy&main-color=#89D6FF; structures, "
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "1876916293": [
+        # Fungal Hall B PB Tip
+        # Replaces MQB lower right computer screen scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;The item within Fungal Hall B's mushroom ring can be uncovered with "
+            "&main-color=#FF3333;Morph Ball Bombs&main-color=#89D6FF; or "
+            "&main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "4186358462": [
+        # Life Grove Tunnel PB Tip
+        # Replaces Life Grove & LGT Bendezium rock scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;This &main-color=#FF3333;Bendezium&main-color=#89D6FF; surface has a note:\n\n\n "
+            "Life Grove and Life Grove Tunnel each contain a &main-color=#FF3333;Sandstone&main-color=#89D6FF; "
+            "structure.\n\nLike most &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures, "
+            "they can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "323204925": [
+        # Watery Hall Access Wall Tip
+        # Replaces Watery Hall Access holobanner scan
```

Comment:
Does the debris not already have a scan node? That would be a more suitable node.

Reply:
Agreed, but if I'm reading PWE right, I don't think it actually does